src/main/java/org/reactome/addlinks/kegg/KEGGReferenceDatabaseGenerator.java

```diff
@@ -90,7 +91,12 @@ public static void generateSpeciesSpecificReferenceDatabases(ReferenceObjectCach
 					try
 					{
 						String speciesURL = KEGG_URL.replace("###SP3###", code + ":");
-						String newDBName = "KEGG Gene ("+speciesName + ")";
+						String newDBName = KEGG_GENE + " (" + speciesName + ")";
+						if (code.equals("mtu") || code.equals("mtv"))
+						{
+							newDBName = KEGG_GENE + " ("+speciesName + " - " + code + ")";
+						}
+
```

Comment:
I _could_ do this, but I am not sure I see a point in making this change.

Reply:
Not a significant advantage.  Just to reduce duplication of the same Strings.  Since the code is just an additional suffix, adding it instead of reassignment seemed like a good way to go.

I just found the assignment followed by a possible immediate reassignment on line 97 a bit unusual.  Another way to go could be one assignment in an if block and the other assignment in an else block.  I just find that easier to follow rather than an overwrite of an existing value.

Like I mentioned, it's not a blocker.  I'll leave it up to you.